src/OWML.Utils/EnumCreator.cs

```diff
@@ -0,0 +1,289 @@
+﻿using HarmonyLib;
+using OWML.Common;
+using OWML.Logging;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Emit;
+
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("OWML.ModLoader")]
+
+namespace OWML.Utils
+{
+    public static partial class EnumUtils
+    {
+        private static readonly HashSet<Type> _flagsTypes = new HashSet<Type>
```

Comment:
`Credits.Platform`, `Credits.CreditsType`, `GhostNode.NodeLayer`, `HazardVolume.HazardType`, `InputUtil.XInputButton`, `InputUtil.ScePadButton`, `Shape.Layer`, and `SignalFrequency` do not have a FlagsAttribute. Is this list for enums that have that attribute, or for enums that have enumeration constants of powers of two? A comment explaining this would be good.

Reply:
It is for enums that use powers of two.